spec/support/request_spec_helper.rb

```diff
@@ -5,7 +5,46 @@ def json
     JSON.parse(response.body)
   end
 
-  def version(v = 'v0.0')
-    "/api/#{v}"
+  def version(ver = 'v0.0')
```

Comment:
@bzwei if you add the support `Dir` from the common gem, you don't need to duplicate this support helper. https://github.com/ManageIQ/service_portal-api/pull/135/files#diff-93830fa29d616f7c87903d08b5b1b29a

Reply:
@syncrou For a more flexible and readable purpose, it's better to keep its own spec help methods.